extensions/2.0/Vendor/EXT_mesh_features/schema/primitive.EXT_mesh_features.schema.json

```diff
@@ -37,8 +37,7 @@
   "anyOf": [
     {
       "required": [
-        "featureIds",
-        "propertyTables"
```

Comment:
See https://github.com/CesiumGS/glTF/pull/26 - I'll cherry-pick the change to that PR as well.